test/bigtest/tests/units/settings/MappingProfilesForm/MappingProfilesForm-test.js

```diff
@@ -51,15 +45,18 @@ describe('MappingProfilesForm', () => {
       handleSubmitSpy.resetHistory();
 
       await mountWithContext(
-        <Paneset>
-          <Router>
-            <MappingProfilesFormContainer
-              initialValues={initialValues}
-              onSubmit={handleSubmitSpy}
-              onCancel={noop}
-            />
-          </Router>
-        </Paneset>,
+        <>
+          <div id="OverlayContainer" />
```

Comment:
The div with id OverlayContainer is as a global container for callouts. There are not these things In case of unit tests - applications are not inited. So this is why we have to add it to check callouts.

Reply:
Ah, yes, sounds good.